composer.json

```
@@ -15,27 +15,27 @@
         }
     ],
     "require": {
-        "php": ">=7.3",
+        "php": "^8.0|^8.1",
```

```suggestion
        "php": "^8.0",
```
Caret o Tilde går på minor så 8.0 tillåter också 8.1. Vill du ha 8.0 och 8.1 endast så kanske * är bättre?

---

I detta fallet gjorde jag en PR baserad på parent `Livijn:master`. 

Får rättighetsproblem när jag försöker pusha lokalt, något du kan undersöka?